java/java-ranger-regression/alarm/Alarm_prop1.yml

```diff
@@ -0,0 +1,8 @@
+format_version: "1.0"
```

Comment:
Please update all yml files to the new format 2.0.

Reply:
Sure, will do.

Reply:
Quick question on that, when I changed the format version from 1.0 to 2.0, benchexec does not run anymore on my machine? is there any update or some setup that I need to make to have it running? Thank you!

Reply:
@dbeyer sent an email on September 27th ... you need to have BenchExec 3.3 to support the new format

Reply:
Thanks, for the reply. It seems that I have 3.2, I thought I was using the most recent version. Is there any guide on how to do the migration smoothly?

Reply:
Oh, good to know. Okay, I'll try that then. Thank you for letting me know!